config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -850,7 +851,7 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
                       ungridded_lbound=1, ungridded_ubound=Ice_ocean_boundary%ice_ncat)
   endif
 
-  if (cesm_coupled) then
+  if (cesm_coupled .and. use_MARBL) then
```

Comment:
Should we remove the `cesm_coupled` part of this block? If a non-CESM model wants to use MARBL, it will need these forcing fields (and presumably `use_MARBL` will be false by default)

Reply:
My only concern with doing this is that the [logic for coupling the multi-category ice fields for MARBL](https://github.com/NCAR/MOM6/pull/345#issuecomment-2747593706) is still within an `if (cesm_coupled) then`. So making this change will only currently allow a non-CESM model to use MARBL _without those fields_.

Happy to do whatever you recommend here.

Reply:
Let's leave it as-is, and I'll file this away as something that will need to be done if another NUOPC-driven model wants to turn on MARBL tracers in MOM6.